Source/Tests/ELFCodeLoader2.h

```diff
@@ -403,6 +406,28 @@ class ELFCodeLoader2 final : public FEXCore::CodeLoader {
       Entrypoint = MainElfEntrypoint;
     }
 
+    if (Is64BitMode()) {
+      // Load VDSO if we can
+      FEX_CONFIG_OPT(ThunkGuestLibs, THUNKGUESTLIBS);
```

Comment:
This should be pass in, not read through the config directly

Reply:
In fact, I think it would be best to pass in (VDSOFD/VDSOSize) directly, or load the VDSO before elf loading and pass the base pointer (though that might lead to address conflicts I guess?)

I don't think ELFLoader and thunks need to be coupled